addon/-private/system/normalize-link.ts

```diff
@@ -1,3 +1,11 @@
+export interface Link {
+  href: string;
+}
+
+export type NormalizedLink =
+| Link
+| null;
```

Comment:
Should use some centrally defined Maybe<T>, rather than defining and exporting a new type alias